website/docs/advanced/stickiness.md

```diff
@@ -0,0 +1,36 @@
+---
+id: stickiness
+title: Stickiness
+---
+
+_Stickiness_ is how Unleash guarantees that the same user gets the same features every time. Stickiness is useful in any scenario where you want to either show a feature to only a subset of users or give users a variant of a feature.
+
+## Calculation
+
+By default, Unleash calculates stickiness based on the user id and the group id. If the user id is unavailable, it falls back to using the session id instead. It hashes these values to a number between 0 and 100. This number is what determines whether a user will see a specific feature or variant. Because the process is deterministic, the same user will always get the same number.
```

Comment:
Do we want to point out that we use the murmur hash, or is that not interesting here?

Reply:
I don't know, actually, but I like the suggestion. I'll add it in! 🙋🏾